src/commandDetails/coin/transfer.ts

```diff
@@ -0,0 +1,122 @@
+import { container } from '@sapphire/framework';
+import { User } from 'discord.js';
+import {
+  CodeyCommandOptionType,
+  CodeyCommandDetails,
+  SapphireMessageExecuteType,
+  getUserFromMessage,
+  SapphireMessageResponseWithMetadata,
+} from '../../codeyCommand';
+import {
+  adjustCoinBalanceByUserId,
+  getCoinBalanceByUserId,
+  UserCoinEvent,
+} from '../../components/coin';
+import { getCoinEmoji } from '../../components/emojis';
+import { pluralize } from '../../utils/pluralize';
+
+const coinTransferExecuteCommand: SapphireMessageExecuteType = async (
+  client,
+  messageFromUser,
+  args,
+) => {
+  // First mandatory argument is receiving user
+  const receivingUser = <User>args['user'];
+  if (!receivingUser) {
+    throw new Error('please enter a valid user mention or ID for balance transfer.');
```

Comment:
Capitalize the error messages.

Reply:
These should also be returning a SapphireMessageResponses and not throwing an error right - to notify the user?